tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -350,6 +363,14 @@ stages:
         SpecificArtifact: ${{ parameters.specificArtifact }}
         BuildId: ${{ parameters.BuildId }}
 
+    - template: flex-downloadPipelineArtifact.yml
+      parameters:
+        StepName: 'Download Android-QNN-aar Pipeline Artifact'
+        ArtifactName: 'onnxruntime-android-qnn-aar'
+        TargetPath: '$(Build.BinariesDirectory)/nuget-artifact'
```

Comment:
is the QNN build being added to the nuget package too?

Reply:
@skottmckay should the package be build only for maven? Edward also mentioned we have NPM packaging